x-pack/plugins/maps/public/routing/routes/list/maps_list_view.js

```diff
@@ -53,17 +52,13 @@ export class MapsListView extends React.Component {
     listingLimit: getUiSettings().get('savedObjects:listingLimit'),
   };
 
-  UNSAFE_componentWillMount() {
-    this._isMounted = true;
-    updateBreadcrumbs();
```

Comment:
No need to update bread crumbs here. `MapsAppView` cleans up breadcrumbs on unmount